src/requestTracing/constants.ts

```diff
@@ -43,6 +43,7 @@ export enum RequestType {
     STARTUP = "Startup",
     WATCH = "Watch"
 }
+export const FAILOVER_REQUEST_TAG = "Failover";

```

Comment:
Could you move this line after line 49, since we have a comment line `//Tag names` in line 49